images/sdn/Dockerfile.rhel

```diff
@@ -4,12 +4,16 @@ COPY . .
 RUN make build --warn-undefined-variables
 RUN CGO_ENABLED=0 GO_BUILD_FLAGS="-tags no_openssl" make build GO_BUILD_PACKAGES="github.com/openshift/sdn/cmd/openshift-sdn-cni" --warn-undefined-variables
 
+FROM registry.ci.openshift.org/ocp/4.13:cli AS cli
 FROM registry.ci.openshift.org/ocp/4.13:base
```

Comment:
`cli` is literally just `base` plus `oc` (and some symlinks to `oc`). It would be more efficient to just change the base image to be `4.13:cli` rather than `4.13:base`. ("More efficient" as in, less data to pull/store per node.)